raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -459,21 +487,17 @@ def set_expired_pairs(transfers_pair, block_number):
 
 
 def events_for_refund_transfer(
-        refund_channel,
-        transfer_to_refund,
-        pseudo_random_generator,
-        block_number,
+        refund_channel: NettingChannelState,
+        transfer_to_refund: LockedTransferSignedState,
+        pseudo_random_generator: random.Random,
+        block_number: typing.BlockNumber,
 ):
     """ Refund the transfer.
     Args:
-        refund_route (RouteState): The original route that sent the mediated
-            transfer to this node.
-        transfer_to_refund (LockedTransferSignedState): The original mediated transfer
+        refund_channel: The original channel through which the mediated
+            transfer to this node was sent.
+        transfer_to_refund: The original mediated transfer
             from the refund_route.
-        timeout_blocks (int): The number of blocks available from the /latest
-            transfer/ received by this node, this transfer might be the
-            original mediated transfer (if no route was available) or a refund
-            transfer from a down stream node.
         block_number (int): The current block number.
```

Comment:
I know it's unrelated, but since you're editing the docstring, could you also remove this redundant type annotation?